buildSrc/src/main/groovy/org.apache.fineract.dependencies.gradle

```diff
@@ -84,7 +84,6 @@ dependencyManagement {
             exclude 'commons-io:commons-io'
             exclude 'commons-logging:commons-logging'
             exclude 'commons-codec:commons-codec'
-            exclude 'org.apache.commons:commons-compress'
```

Comment:
Do we need this package for something?

Reply:
It is being excluded, in the original file, exclusion is declared twice. I have removed one of them to avoid duplicated lines.